src/main/java/com/google/maps/GeoApiContext.java

```diff
@@ -62,49 +83,40 @@
    * @see GaeRequestHandler
    */
   public interface RequestHandler {
-    <T, R extends ApiResponse<T>> PendingResult<T> handle(String hostName, String url, String userAgent, Class<R> clazz,
-                                                          FieldNamingPolicy fieldNamingPolicy, long errorTimeout,
-                                                          Integer maxRetries,
-                                                          ExceptionsAllowedToRetry exceptionsAllowedToRetry);
-
-    <T, R extends ApiResponse<T>> PendingResult<T> handlePost(String hostName, String url, String payload,
-                                                              String userAgent, Class<R> clazz,
-                                                              FieldNamingPolicy fieldNamingPolicy, long errorTimeout,
-                                                              Integer maxRetries,
-                                                              ExceptionsAllowedToRetry exceptionsAllowedToRetry);
-
-    void setConnectTimeout(long timeout, TimeUnit unit);
-    void setReadTimeout(long timeout, TimeUnit unit);
-    void setWriteTimeout(long timeout, TimeUnit unit);
-    void setQueriesPerSecond(int maxQps);
-    @Deprecated void setQueriesPerSecond(int maxQps, int minimumInterval);
-    void setProxy(Proxy proxy);
-  }
 
-  private long errorTimeout = DEFAULT_BACKOFF_TIMEOUT_MILLIS;
+    <T, R extends ApiResponse<T>> PendingResult<T> handle(String hostName, String url,
+        String userAgent, Class<R> clazz,
+        FieldNamingPolicy fieldNamingPolicy, long errorTimeout,
+        Integer maxRetries,
+        ExceptionsAllowedToRetry exceptionsAllowedToRetry);
 
-  /**
-   * Construct a GeoApiContext with OkHttp.
-   */
-  public GeoApiContext() {
-    this(new OkHttpRequestHandler());
-  }
+    <T, R extends ApiResponse<T>> PendingResult<T> handlePost(String hostName, String url,
+        String payload,
+        String userAgent, Class<R> clazz,
+        FieldNamingPolicy fieldNamingPolicy, long errorTimeout,
+        Integer maxRetries,
+        ExceptionsAllowedToRetry exceptionsAllowedToRetry);
+
+    interface Builder {
+
```

Comment:
I hit the whole code base with [google-java-format](https://github.com/google/google-java-format) to bring it into line with our code format standards. I can do that as a separate PR on the master branch and regnerate this branch to bring the noise down.